pkg/config/config.go

```diff
@@ -89,3 +96,19 @@ func validateAdditionalResources(resources []string) error {
 	}
 	return nil
 }
+
+func validateTargetVersion(version string) error {
```

Comment:
consider a separate file for this? with a test?

Reply:
Tests are coming 👍 I would keep it here for now, same as the other `validate*`, can rearrange later, maybe move all of those.